src/client/automation/utils/key-identifier-required-for-event.js

```diff
@@ -0,0 +1,7 @@
+import hammerhead from '../deps/hammerhead';
+
+const browserUtils = hammerhead.utils.browser;
+
+/* eslint-disable no-undef */
```

Comment:
I've got "'keyIdentifierRequiredForEvent' is not defined" message when linting without this comment